app/editor/src/features/admin/filters/constants/searchInOptions.ts

```diff
@@ -0,0 +1,22 @@
+export const searchInOptions = (callback: (value: string | number | undefined) => void) => [
+  {
+    id: 'all',
+    label: 'All',
+    onClick: () => callback('all'),
```

Comment:
I'm not a fan of the implementation of the `ToggleGroup` requiring event hooks to each button instead of the group.